supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -309,6 +315,15 @@ async def refresh_session(
                 response["accessToken"]["token"]
             ).payload
 
+            new_refresh_token: Optional[Dict[str, Any]] = response.get("refreshToken")
+            new_refresh_token_info: Optional[TokenInfo] = None
+            if new_refresh_token is not None:
```

Comment:
When could this be `None`?

Reply:
Shouldn't unless something is wrong with core. 

So it should throw error. Updated accordingly. If refreshToken key isn't present, it will raise KeyError which will get handled.